src/app/static/src/tests/components/header/fileMenu.test.ts

```diff
@@ -127,14 +127,14 @@ describe("File menu", () => {
                 programme: {hash: "5csv", filename: "5.csv"},
                 anc: {hash: "6csv", filename: "6.csv"}
             }
-        });
+        };
 
         const actualBlob = (mockCreateObjectUrl as jest.Mock).mock.calls[0][0];
         const reader = new FileReader();
         reader.addEventListener('loadend', function () {
             const text = reader.result as string;
             const result = JSON.parse(text)[1];
-            expect(result).toStrictEqual(expectedJson);
+            expect(JSON.parse(result)).toStrictEqual(expectedJson);
```

Comment:
This test started failing because it was acutally doing a string compare rather than JSON compare, and we have modified the order of the keys somewhat in saved state. This change is to do JSON compare instead.